### ISSUE TITLE
logstash 5.0 errors out when trying to get "--help" page

### ISSUE THREAD

**Issue by DreadPirateRob** [elastic/logstash#5722]
Trying to get the help page of the `logstash` command so that I can figure out what the f is going on b/c my logstash isn't working. 

I can't because if I don't specify the `--path.settings` as an argument, I get an error stating that its needed. This is kind of a bad implementation of the `--path.settings` argument. See below. 

- Version:
```
[root@logstash ~]# yum list installed | grep -E '(elasticsearch|logstash)'
elasticsearch.noarch   5.0.0_alpha4-1   @elasticsearch-5.x                      
logstash.noarch        1:5.0.0~alpha4-1 @logstash-5.0 
```
- Operating System:
```
[root@logstash ~]# cat /etc/redhat-release 
CentOS release 6.8 (Final)
```
- Java Version:
```
[root@logstash ~]# java -version 
openjdk version "1.8.0_101"
OpenJDK Runtime Environment (build 1.8.0_101-b13)
OpenJDK 64-Bit Server VM (build 25.101-b13, mixed mode)
```

- Config File (if you have sensitive info, please remove it):
- Sample Data:
- Steps to Reproduce:

So I was trying to get the verbose on the screen so I can see why logstash isn't starting up, so I did this: 
```
[root@logstash logstash]# /usr/share/logstash/bin/logstash -v -f logstash-remotes.conf --path.settings /etc/logstash
--- jar coordinate com.fasterxml.jackson.core:jackson-annotations already loaded with version 2.7.1 - omit version 2.7.0
--- jar coordinate com.fasterxml.jackson.core:jackson-databind already loaded with version 2.7.1 - omit version 2.7.1-1
ERROR: Unrecognised option '-v'

See: 'bin/logstash --help'

```
Well dang, so I guess I'll check the help page. 
So I did the command as suggested: 
```
[root@logstash logstash]# /usr/share/logstash/bin/logstash --help
--- jar coordinate com.fasterxml.jackson.core:jackson-annotations already loaded with version 2.7.1 - omit version 2.7.0
--- jar coordinate com.fasterxml.jackson.core:jackson-databind already loaded with version 2.7.1 - omit version 2.7.1-1
Logstash has a new settings file which defines start up time settings. This file is typically located in $LS_HOME/config or /etc/logstash. If you installed Logstash through a package and are starting it manually please specify the location to this settings file by passing in "--path.settings=/path/.." in the command line options {:level=>:warn}
Failed to load settings file from "path.settings". Aborting... {"path.settings"=>"/usr/share/logstash/config", "exception"=>Errno::ENOENT, "message"=>"No such file or directory - /usr/share/logstash/config/logstash.yml", :level=>:fatal}
[root@logstash logstash]# 

```

So that didn't work. What's the deal y'all? 

btw, I'm doing all of this just to figure out why logstash isn't working. It doesn't appear to actually be starting when I do `initctl start logstash`. I say that because the `/var/log/logstash/logstash.log` file is empty.

**Comment by suyograo** [comment]
@DreadPirateRob ack. `--help` shouldn't care if `path.settings` is not set. Will fix.

**Comment by jsvd** [comment]
fixed in #5786
